packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -209,11 +199,8 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         ),
     ];
 
-    return StreamProvider<Screen>(
-      create: (BuildContext context) {
-        return _screenController.stream;
-      },
-      initialData: _currentScreen,
+    return ValueListenableProvider<Screen>.value(
```

Comment:
nit: omit the generic <Screen>
Is there any reason to use 
`ValueListenableProvider.value`
instead of
`ValueListenableBuilder`
?
They seem like they are identical but one is a shorter name and defined in package:flutter.

Reply:
I believe that one just provides the value to widgets further down the tree, and the other will rebuild the sub-tree if the value changes here (here, we want the former but not the later).

Reply:
When widgets further down the tree use the value, they'll be re-built automatically when the value changes (for this case, it's the status line widget).

Reply:
ok I get it now. That is more convenient.